Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
CoZ45Ugbv686ztOvtXC4tReA/OwMVpcFqCgNf1SWBlhakk9Opt/jxJIuUrrImZkfeA24BqgDXgHe55yrPtNrVOTkfA0HQ7T1DXKiJ/LRO0BrzyANHSepbevjWFsftW199A6+vndVSUE2FaUB1s+bzpsWF7Fh/gz9h0AkgTV39bPlcBsvH25jT0Mnexu7OTn0xv9Nzy/KZX5RLuVFuRQXZFOcn8XM/Gxm5WczKz+L/OwMTY+Q83amIpcqEwA2Awecc4cAzOwB4GbgjEUu3vY1ddPWOwiAY0xZnuDLsb167GtHPz62go8t5e6MX7zxfU/7nhPmm+B7nPbaaLJHke+01575j5GxDwVDjqFgiKGQY2g4xHAoxFAwfG048u/QqX9D9A4G6RsYpmdgmL7BIL0Dw/QODtM7EKRnYHjc75mT6WN+US7zZuRy0eKZzCvKZfnsfCpKA8zKzz5jVhFJPCWBHG5aN5eb1s0Fwr9Djp7opaaxm8OtPRyL/NG25XAbv9pRf9rvHACfhe/oFeRkkp+dQX5OBvnZGeRl+8nO8JPl95GVMeoj8nV2ho9Mvw+fz/Cb4TPw+Qxf5HO/zzAb/zGfGYzqjmNr5Ohiefpjo19nEzw29oXjvy4dOuyG+TPIyvBm6D1VilwZUDvq6zrgTR5lAeArv9/H49XHvYwgUfJZ+Hif8IeR6feRl51BbpafvOwMZuVnMX9mLvlZGeRm+wnkZDIr8pf3zLyRv8CzKJyWqb++RVKU32csLs4fdxrEcDBEW+8gLT3hu/Ot3QO09gzQ3R/+YzD87xA9A8N09A1S1z7MYDDE4PCoj8gfk5Jctn7+as/+UE+VIjfefzVP+1+Cmd0B3AEwf/78uAb6m7ct5yOXLBr1vcdkOT1bFM8986MTvXZsuXjjY2NfN/H7Tvw9J/hZJvmX3tnyTfRzT/R/25GyluE3svw+MkZKm8+nrT1E5Lxk+H2UBHIoCZzfQqVQyDEYDDEwHB4ZCDmHc+G7gWM/D3+M+jrEqesj4jFyMtHozBvfIz1KaSAn07PvnSpFrg6YN+rrcqBh7JOcc98BvgPhOXLxDKQdw0VE5Fz4fEaOz6/5szIpqbKW+hVgmZktMrMs4FbgIY8ziYiIiMRVStyRc84Nm9lfAY8R3n7k+865Ko9jiYiIiMRVShQ5AOfcw8DDXucQERERmSqpMrQqIiIiknZU5ERERESSlIrc/9/eI31UEwAAC7ZJREFUnQfNUdRhHP8+lYCccoUjcmgobqtQAlKggrFADEFAFAGliiCCoFKAVWJAriACBQgWIIdccoiAyCEiyCEESoFwmQuTkAQCxMSEkCJAARHk5x/dq+O6m3ffvNljdp9P1dbO9vT0dm/Pu/N7u2d2zMzMzErKgZyZmZlZSTmQMzMzMyspB3JmZmZmJeVAzszMzKykHMiZmZmZlZQDOTMzM7OSciBnZmZmVlIO5MzMzMxKyoGcmZmZWUkpItpdh7aQ9CrwUpPfZgiwsMnv0cl6uf1ue+/q5fb3ctuht9vfy22H1rT/oxGxbnVizwZyrSDp6YjYod31aJdebr/b3ptth95ufy+3HXq7/b3cdmhv+z21amZmZlZSDuTMzMzMSsqBXHNd0e4KtFkvt99t71293P5ebjv0dvt7ue3Qxvb7HDkzMzOzkvKInJmZmVlJOZAzMzMzKykHcgMk6WuSnpP0gaQdqtadKGmmpOmSvlhn+7UlPSBpRn5eqzU1X74k3SJpQn7MljShTr7ZkibnfE+3up7NImmspL8XPoNRdfKNzPvDTEkntLqezSDpPEnTJE2SdIekNevk65q+76sflVyU10+SNLwd9WwGSRtLeljS1Pzdd2yNPCMkLS78PZzajro2S1/7crf2v6QtC306QdIbko6rytNVfS/pGkkLJE0ppDV03G7Z931E+DGAB7A1sCUwDtihkL4NMBH4EDAMmAUMqrH9ucAJefkE4Jx2t2k5fCbnA6fWWTcbGNLuOjahzWOBH/SRZ1DeDzYFVsz7xzbtrvtyaPsewOC8fE69fbhb+r6RfgRGAfcCAnYCxre73sux/UOB4Xl5deD5Gu0fAdzd7ro28TNY6r7czf1faOMg4B+kH6nt2r4HdgWGA1MKaX0et1v5fe8RuQGKiKkRMb3Gqn2BmyNiSUS8CMwEdqyT77q8fB3w5ebUtDUkCTgAuKnddelAOwIzI+KFiPgncDOp/0stIu6PiPfzyyeAjdpZnxZopB/3Ba6P5AlgTUlDW13RZoiIeRHxbF5+E5gKbNjeWnWcru3/gt2AWRHR7DsktVVEPAosqkpu5Ljdsu97B3LNsyHwSuH1HGp/2a0fEfMgfUEC67Wgbs20CzA/ImbUWR/A/ZKekfTtFtarFY7O0yjX1Blqb3SfKLPDSCMRtXRL3zfSj73Q10j6GLAdML7G6p0lTZR0r6SPt7RizdfXvtwL/X8Q9f9h7+a+h8aO2y3bBwY3o9BuI+lBYIMaq06KiN/V26xGWql/66XBz+HrLH007jMRMVfSesADkqbl/3g63tLaD1wGnEHq4zNI08uHVRdRY9tS7BON9L2kk4D3gRvrFFPavq/SSD+Wtq8bJWk14DbguIh4o2r1s6Qpt7fy+aJ3Apu3uo5N1Ne+3NX9L2lFYB/gxBqru73vG9WyfcCBXAMiYvdl2GwOsHHh9UbA3Br55ksaGhHz8tD7gmWpYyv09TlIGgx8Bdh+KWXMzc8LJN1BGn4uxcG80f1A0pXA3TVWNbpPdJwG+n408CVgt8gniNQoo7R9X6WRfixtXzdC0gqkIO7GiLi9en0xsIuIeyRdKmlIRHTFTdUb2Je7uv+BPYFnI2J+9Ypu7/uskeN2y/YBT602z13AQZI+JGkY6T+SJ+vkG52XRwP1RvjKYHdgWkTMqbVS0qqSVq8sk06Sn1Irb9lUnf+yH7Xb9RSwuaRh+T/ag0j9X2qSRgJjgH0i4u06ebqp7xvpx7uAQ/LVizsBiytTMWWXz4O9GpgaERfU
```

Comment:
I think this value won't give the probability you calculated above. Maybe just use `stats.norm.ppf` to get the signal value